app/controllers/community_resources_controller.rb

```diff
@@ -17,7 +17,18 @@ def new
   end
 
   def create
-    community_resource.assign_attributes permitted_attributes(community_resource)
+    location_params = params['community_resource']['location']
```

Comment:
If Location is not required on a new Community Resource, I will need to add some checks in place here (currently required form fields).